pymc/tests/distributions/util.py

```diff
@@ -317,10 +317,11 @@ def _model_input_dict(model, param_vars, pt):
         pt_d = _model_input_dict(model, param_vars, pt)
         pt_logp = pm.Point(pt_d, model=model)
         pt_ref = pm.Point(pt, filter_model_vars=False, model=model)
-        npt.assert_almost_equal(
```

Comment:
The same changes should be done to `check_logcdf` and `check_selfconsistency_discrete_logcdf`